cluster/examples/kubernetes/ceph/cluster.yaml

```diff
@@ -20,7 +20,7 @@ spec:
     # v12 is luminous, v13 is mimic, and v14 is nautilus.
     # RECOMMENDATION: In production, use a specific version tag instead of the general v14 flag, which pulls the latest release and could result in different
     # versions running within the cluster. See tags available at https://hub.docker.com/r/ceph/ceph/tags/.
-    image: ceph/ceph:v14.2.1-20190430
+    image: ceph/ceph:latest-nautlius-devel
```

Comment:
the repo name and the tag name are incorrect.

It's ceph/daemon-base:latest-nautilus-devel